flexible_event_config.md

```diff
@@ -169,6 +170,15 @@ When the `event_report_window` for a spec completes, we will map its summary val
   "trigger_summary_bucket": [<bucket start>, <bucket end>]
 }
 ```
+### Trigger prioritization
+
+Given that triggering attribution can affect a source's state without producing a report, we will need a new algorithm for doing trigger prioritization. Here is a sketch of how it could work:
+
+1. Always maintain a sorted list of triggers, sorted in order of priority (descending) and trigger time (ascending)
+2. Whenver a spec's window's end time is hit (breaking ties arbitrarily)
+    1. Iterate through triggers in order, "applying" them to generate a list of "speculative" reports. Stop when privacy limits are hit.
```

Comment:
limits are hit OR all of the triggers are consumed.

Reply:
"Iterate through triggers in order" already encompasses the second part of that disjunction.

Reply:
> "Iterate through triggers in order" already encompasses the second part of that disjunction.

Yes. You are right. I just suggest in writing to emphasize there are two outcomes for final states: 1, report limited reached, 2 no more triggers

Reply:
I don't think this is necessary

Reply:
> I don't think this is necessary, lmk if you feel strongly

I am OK. No strong opinion. Just wording.